content/app/app-dev-course/modul5/_index.en.md

```diff
@@ -82,7 +82,7 @@ Update the policy file so that it's authorization rules cover the new process st
 ### Useful documentation
 - [Custom validation](/app/development/logic/validation/#how-to-add-custom-validation)
 
-### Comprehension check
+### Knowledge check
```

Comment:
Overrule standard texts for confirmation page
Kan vi heller bruke replace standard texts ? Vil anta uttrykket benytes flere steder i oversettelsene.

Reply:
Linje 97: kan kanskje ha norsk tekst her også ? Begynner å bli usikker på om det var rett fremgangsmåte å skulleha alle tekstene på engelsk, så du kan velge selv. men fint om vi da er konsekvente hele veien. Tenker infosiden og kan være på samme språk som tekstene uavhengig av hvilken vei vi får.